test/сondensed_matter_physics/effective_mass_of_the_electron_from_the_energy_test.py

```diff
@@ -0,0 +1,41 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+
+from sympy import pi
+from sympy.physics.units import planck as planck_constant
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    dimensionless,
+    Quantity,
+    SI,
+)
+from symplyphysics.laws.сondensed_matter_physics import effective_mass_of_the_electron_from_the_energy as ef_mass_el
+
+# Description
+## Consider the case of the quadratic isotropic law of dispersion of charge carriers.
+
+
+@fixture(name="test_args")
+def test_args_fixture():
+    propogation_vec_axis = Quantity(2e8 * pi * (dimensionless/units.length))
+    energy_func = ((planck_constant/2/pi)**2) * (propogation_vec_axis**2) / 2 / Quantity(1 * units.mass) / 2.1218e-11
```

Comment:
It looks quite synthetic. You simply copied the law so it would result in expected value. Try to find some actual example for this law.